File: Heat/ViewModels/Plants/AddMediumPlantViewModel.vb
Imports System.ComponentModel.DataAnnotations
Imports Heat.Models

Namespace ViewModels.Plants
    Public Class AddMediumPlantViewModel

        Property PlantId As Integer

        <Required>
        <Display(Name:="Descrizione immagine")>
        Property Description As String

        <Display(Name:="Etichette")>
        Property Tags As String

        <Display(Name:="File")>
        <DataType(DataType.Upload)>
        Property UploadFile As HttpPostedFileBase

    End Class

End Namespace

File: Heat.ConvertedToC#/ViewModels/Plants/AddMediumPlantViewModel.cs
using System.Web;
using System.ComponentModel.DataAnnotations;

namespace Heat.ViewModels.Plants
{
    public class AddMediumPlantViewModel
	{

		public int PlantId { get; set; }

		[Required()]
		[Display(Name = "Descrizione immagine")]
		public string Description { get; set; }

		[Display(Name = "Etichette")]
		public string Tags { get; set; }

		[Display(Name = "File")]
		[DataType(DataType.Upload)]
		public HttpPostedFileBase UploadFile { get; set; }

	}

}
